server/platform/shared/filestore/s3store.go

```diff
@@ -178,7 +186,8 @@ func (b *S3FileBackend) s3New(isCloud bool) (*s3.Client, error) {
 	}
 
 	if b.trace {
-		s3Clnt.TraceOn(os.Stdout)
+		s3Trace := s3Trace{}
```

Comment:
It's weird to have variable name and struct name as the same. Let's name the variable as `tracer` or sth similar.

Reply:
This seems fair, my bad.